### ISSUE TITLE
Libvlc initialization segfaults on windows 8.1 pro

### ISSUE THREAD

**Issue by whirm** [Tribler/tribler#1618]
```
Traceback (most recent call last):
  File "wx\_misc.pyo", line 1358, in Notify
    
  File "wx\_core.pyo", line 14771, in Notify
    
  File "Tribler\Core\Video\VLCWrapper.pyo", line 64, in _init_vlc
    
  File "Tribler\Core\Video\VLCWrapper.pyo", line 22, in invoke_func
    
  File "Tribler\Core\Video\VLCWrapper.pyo", line 160, in get_vlc_mediactrl
    
  File "Tribler\vlc.pyo", line 4627, in libvlc_media_player_new
    
WindowsError: exception: access violation reading 0x00000000
```

**Comment by LipuFei** [comment]
@whirm I have been running vlc streaming on Win10 for a while, haven't seen this error yet.

**Comment by whirm** [comment]
This happens at startup time. (it also comes from the big log)

**Comment by LipuFei** [comment]
Should be fixed by #1642
